koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -533,10 +542,15 @@ def sigint_handler(*args, **kwargs):
 
         if self.osbs().build_was_cancelled(build_id):
             self.session.cancelTask(self.id)
+            self._upload_logs_once()
             raise ContainerCancelled('Image build was cancelled by OSBS.')
 
         elif not has_succeeded:
-            error_message = self.osbs().get_build_error_message(build_id)
+            error_message = None
+            try:
+                error_message = self.osbs().get_build_error_message(build_id)
+            except Exception as ex:
+                self.logger.error("Error during getting error message: %s", repr(ex))
```

Comment:
Is worth to log it as `logger.exception` to keep traceback for debugging?

Reply:
doesn't hurt